### PR TITLE
Prettier

### LINKED ISSUE #71: Introduce automated code formatting
While working on the PR for the improved matcher messages, I've noticed that there isn't any code formatting tool setup in this repository.  Would you be opposed to introducing a tool such as Prettier to help with code consistency, readability, and ease of development?

### PR DESCRIPTION
**Pull request by mskelton**
Fixes #71 

@mxschmitt In #71 you mentioned wanting to use the default settings of Prettier.  The only except to that in this PR is semicolons.  I observed that the majority of this repo doesn't use semis, so I turned off that rule.  If you would like me to leave it fully default, totally fine but it will increase the diff a little bit.

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
-    - run: npm test
-    - run: npx codecov
-      env:
-        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          # chromium dependencies
+          sudo apt-get install libgbm1
```
Comment (mxschmitt) [inline .github/workflows/nodejs.yml:16]: drive-by:
```suggestion
          npx playwright-chromium install-deps chromium
```

**Review by mxschmitt — APPROVED** [review]

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (mskelton) [reply .github/workflows/nodejs.yml:16]: Neat
